### ISSUE TITLE
Plex for Android TV throwing exceptions on playMedia calls

### ISSUE THREAD

**Issue by kudos** [d8ahazard/Phlex#280]
#### 6. What are the last six digits of your server's API Token? (Settings -> Phlex)
`0ab776`

Plex on my Nvidia Shield is showing the following error in the logs when I try to get Phlex to tell it to play media. I'm guessing a Plex update is causing this, but it might be that the API isn't quite being used correctly, or perhaps some parameters changed.

```12-28 18:01:50.456  i: [pms] /10.0.1.2:59372 - POST /player/playback/playMedia
12-28 18:01:50.456  i: [Remote Control] Waking up device
12-28 18:01:50.459  e: An exception occurred: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference
12-28 18:01:50.459  e: Stacktrace: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference
	at com.plexapp.plex.net.pms.ab.a(SourceFile:470)
	at com.plexapp.plex.net.pms.ab.a(SourceFile:448)
	at com.plexapp.plex.net.pms.ab.a(SourceFile:160)
	at com.plexapp.plex.net.pms.v.a(SourceFile:62)
	at org.jboss.netty.channel.as.b(SourceFile:80)
	at org.jboss.netty.channel.aa.a(SourceFile:783)
	at org.jboss.netty.handler.b.d.b(SourceFile:149)
	at org.jboss.netty.channel.aa.a(SourceFile:783)
	at org.jboss.netty.handler.codec.http.g.a(SourceFile:116)
	at org.jboss.netty.channel.as.b(SourceFile:80)
	at org.jboss.netty.channel.aa.a(SourceFile:783)
	at org.jboss.netty.channel.v.a(SourceFile:302)
	at org.jboss.netty.handler.codec.replay.a.a(SourceFile:527)
	at org.jboss.netty.handler.codec.replay.a.a(SourceFile:506)
	at org.jboss.netty.handler.codec.replay.a.a(SourceFile:443)
	at org.jboss.netty.channel.as.b(SourceFile:80)
	at org.jboss.netty.channel.v.a(SourceFile:274)
	at org.jboss.netty.channel.v.a(SourceFile:261)
	at org.jboss.netty.channel.socket.nio.o.b(SourceFile:351)
	at org.jboss.netty.channel.socket.nio.o.a(SourceFile:282)
	at org.jboss.netty.channel.socket.nio.o.run(SourceFile:202)
	at org.jboss.netty.util.h.run(SourceFile:108)
	at org.jboss.netty.util.internal.x$1.run(SourceFile:44)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
	at java.lang.Thread.run(Thread.java:761)
```

**Comment by kudos** [comment]
Even if this is not something that can be fixed in Phlex, Phlex should at least recognise the 500 status code that Plex returns and treat it as a failure not a success.

**Comment by kudos** [comment]
I found the issue, this line has a blank value for `$serverPort` in the my logs. Manually adding the port makes it work. I'm going to see if I can hack in a fix.

**Comment by d8ahazard** [comment]
Fixed with #281
